var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,106 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently not supported
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
```

Comment:
I remember that some older versions on debian/ubuntu needed a `-pthread` patch in some of our systems (https://github.com/ornladios/ADIOS/pull/68).

maybe I should only add `1.9.0` and `1.10.0` here.